app/views/admin/editions/show/_main.html.erb

```diff
@@ -249,7 +249,10 @@
       <% end %>
     </section>
   <% end %>
-  <% if @edition.translatable? %>
+
+  <% if Flipflop.document_hub? %>
+    <%= render Admin::Editions::Show::TranslationsSummaryCardComponent.new(edition: @edition) %>
+  <% elsif @edition.translatable? %>
```

Comment:
Personally I'd do the `translatable?` check outside of the feature flag check and then not need to do it again inside the view component, but I can see why you've gone for this option with a view towards removing the feature flag